packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -1034,6 +1055,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
                     : null,
             suffixIcon: trailingButton,
           ).applyDefaults(effectiveInputDecorationTheme),
+          maxLines: widget.maxLines,
```

Comment:
Same nit as the other PR. We can move these one-line assignments above the `style:...`. Just feel it would be a little bit easier to maintain:)

Reply:
Done